### ISSUE TITLE
[accessibility]: add the missing aria role to the permalinks on the docs and blog

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1735]
The following Accessibility warning from webhints must be fixed 

![Screen Shot 2022-03-08 at 11 55 38 PM](https://user-images.githubusercontent.com/46004116/157306086-c8848fd9-5cbc-4490-a1ad-8331101d17d4.png)

The solution might be to add a role attribute to the element (not 100% sure need to re-search this).

**Comment by Siddhant-K-code** [comment]
It's already resolved now.
